source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -2063,7 +2063,7 @@ PyAttributeDef KX_GameObject::Attributes[] = {
 	KX_PYATTRIBUTE_RW_FUNCTION("debug",	KX_GameObject, pyattr_get_debug, pyattr_set_debug),
 	KX_PYATTRIBUTE_RO_FUNCTION("components", KX_GameObject, pyattr_get_components),
 	KX_PYATTRIBUTE_RW_FUNCTION("debugRecursive",	KX_GameObject, pyattr_get_debugRecursive, pyattr_set_debugRecursive),
-	KX_PYATTRIBUTE_RO_FUNCTION("lod", KX_GameObject, pyattr_get_lodlist),
+	KX_PYATTRIBUTE_RO_FUNCTION("lodManager", KX_GameObject, pyattr_get_lodmanager),
```

Comment:
KX_LodManager is a very nice name for the internal source but as for `filterManager`i don't know if the user like it or find too long ? @Maujoe: What's your opinion ?